### PR TITLE
[CosmosDB] Managed Cassandra Hotfix

### PR DESCRIPTION
**Pull request by AverageDesigner**
cli extension module version 0.8.0 introduced regression because of python sdk update. It is being fixed in this PR.

### CODE REVIEW

File: src/cosmosdb-preview/azext_cosmosdb_preview/custom.py [hunk 1]
```diff
@@ -566,7 +567,11 @@ def cli_cosmosdb_managed_cassandra_datacenter_create(client,
         base64_encoded_cassandra_yaml_fragment=base64_encoded_cassandra_yaml_fragment
     )
 
-    return client.begin_create_update(resource_group_name, cluster_name, data_center_name, data_center_properties)
+    data_center_resource = DataCenterResource(
+        data_center_properties=data_center_properties
```
Comment (MehaKaushik) [inline src/cosmosdb-preview/azext_cosmosdb_preview/custom.py:571]: Is this a bug in sdk generation or was something missed in Swagger ?? Did you make changes to the sdk manually ?? Ideally we don't want to do that, we should always fix in swagger and generate sdk from swagger.

**Review by frrossms — APPROVED** [review]

File: src/cosmosdb-preview/azext_cosmosdb_preview/custom.py (hunk 1 above)
Comment (frrossms) [reply src/cosmosdb-preview/azext_cosmosdb_preview/custom.py:571]: We haven't changed the Swagger, so it has to be a bug in SDK generation.

**Review by frrossms — COMMENTED** [review]
Are there changes for the nodetool commands that need to be added?

**Comment by yonzhan** [comment]
CosmosDB

**Review by frrossms — APPROVED** [review]

**Review by evelyn-ys — COMMENTED** [review]
Pls update `setup.py` and `history.rst` to upgrade version

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**